integration-tests/src/test/resources/domain5.yaml

```diff
@@ -4,7 +4,6 @@
 adminServerName: admin-server
 domainUID: domain5
 clusterName: cluster-1
-clusterType: CONFIGURED
```

Comment:
Why remove this property instead of change it to DYNAMIC?
It looks to me, clusterType should be changed to DYNAMIC in domain5,  don't be deleted from domain6 and domain9.

Reply:
The clusterType property does not existing in the inputs file any more.